docs/API.md

```diff
@@ -557,32 +673,49 @@ context than inside the listener callback handler, the value is undefined.
 
 ---
 
-| OptName               | Since | Binding | Type   | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ------ | ------ | -------- | ------ |
-| `SRTO_GROUPSTABTIMEO` |       | pre     | `int`  | ms     | 40       | 10+    |
+| OptName               | Since | Binding | Type       | Units  | Default  | Range  | Dir | Entity |
+| --------------------- | ----- | ------- | ---------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_GROUPSTABTIMEO` |       | pre     | `int32_t`  | ms     | 40       | 10-... | W   | GSD+   |
 
 - This setting is used for groups of type `SRT_GTYPE_BACKUP`. It defines the stability 
 timeout, which is the maximum interval between two consecutive packets retrieved from 
-the peer on the currently active link.
-  
-- NOTE: The two packets can be of any type, but this setting usually refers to control
-packets while the agent is a sender. Idle links exchange only keepalive messages once 
-per second, so they do not count. 
-
-- The value of `SRTO_GROUPSTABTIMEO` should be set as small as possible. Keep
-in mind that the time between two consecutive ACK messages (default = 10 ms)
-will vary, hence 40ms is the recommended setting. Note that higher values
-increase the latency penalty. When this time is exceeded, and the link is
-therefore considered unstable, one of the idle links gets activated, at which
-point all packets unacknowledged so far are sent first.
+the peer on the currently active link. These two packets can be of any type,
+but this setting usually refers to control packets while the agent is a sender.
+Idle links exchange only keepalive messages once per second, so they do not
+count. Note that this option is meaningless on sockets that are not members of
+the Backup-type group.
+
+- This value should be set with a thoroughly selected balance and correspond to
+the maximum stretched response time between two consecutive ACK messages. By default
+ACK messages are sent every 10ms (so this interval is not dependent on RTT at all),
```

Comment:
(so this interval is not dependent on RTT at all)

IANAE, but this statement seems contentious to me, and, in any case, is not necessary in the description of ACK intervals. Maybe remove it?

Reply:
I think this explanation is necessary for people to properly configure this option.

It could be written differently, but this actually explains what this option is about. Maybe you'll have some idea how to describe it.

The situation is that ACK messages are sent with 10ms interval:
|    |    |    |    |    |    |    |

But the network jitter may make them at the delivery seen this way:
|    |            | | |  |         ||

This large space after the second bar is the highly stretched response time, which should be still understood as that the link remains maintained. Therefore `groupstabtimeo` option should be set to a value that exceeds the most possibly stretched consecutive response time. If this response time is exceeded, the link will be considered unstable (that is: the group takes a precaution that this link might indeed be broken, and therefore activates a backup link immediately).

Reply:
Proposed rewrite of lines 680 to 694:
```

This setting is used for groups of type `SRT_GTYPE_BACKUP`. It has no effect on 
sockets that are not members of the Backup-type group. Idle links exchange only 
keepalive messages once per second, so they do not count.

- The value of `SRTO_GROUPSTABTIMEO` defines the stability timeout, which is the 
maximum acceptable interval between two consecutive packets retrieved from a 
peer on the currently active link. These two packets can be of any type, but this 
setting usually refers to control packets. For example, the default interval between 
ACK messages is 10ms. However, network jitter may increase the actual interval, 
sometimes by multiples of the default. `SRTO_GROUPSTABTIMEO` can be used to 
account for such increases in determining link stability. If this response time is 
exceeded, the link will be considered unstable. The group responds to the 
potentially broken link by immediately activating a backup link. Note that the 
value of `SRTO_GROUPSTABTIMEO` must be carefully chosen in consideration of the 
consequences of between being too high or too low: 

```